src/preferences/dialog/dlgprefeffectsdlg.ui

```diff
@@ -43,46 +43,40 @@
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
-        <widget class="QTableView" name="availableEffectsList">
-         <property name="maximumSize">
-          <size>
-           <width>400</width>
-           <height>16777215</height>
-          </size>
-         </property>
-         <property name="styleSheet">
-          <string notr="true"/>
-         </property>
-         <property name="selectionMode">
-          <enum>QAbstractItemView::SingleSelection</enum>
-         </property>
-         <property name="selectionBehavior">
-          <enum>QAbstractItemView::SelectRows</enum>
-         </property>
-         <property name="showGrid">
-          <bool>true</bool>
-         </property>
-         <property name="gridStyle">
-          <enum>Qt::NoPen</enum>
-         </property>
-         <attribute name="horizontalHeaderHighlightSections">
-          <bool>true</bool>
-         </attribute>
-         <attribute name="horizontalHeaderStretchLastSection">
-          <bool>true</bool>
-         </attribute>
-         <attribute name="verticalHeaderVisible">
-          <bool>false</bool>
-         </attribute>
-         <attribute name="verticalHeaderHighlightSections">
-          <bool>false</bool>
-         </attribute>
-        </widget>
+        <layout class="QGridLayout" name="effectsTablesGridLayout">
+         <item row="2" column="1">
+          <widget class="QTableView" name="hiddenEffectsTableView"/>
```

Comment:
There are a lot of empty lines in the QTableView: 
![2020-06-09-101528_943x925_scrot](https://user-images.githubusercontent.com/1834516/84124047-f85eaa00-aa3a-11ea-8fda-03ede18c7491.png)

Reply:
I can't reproduce this.

Reply:
Oh, I see. When you remove an effect from the visible effect lists, then close the preferences, then reopen the preferences, there are empty rows. But they go away after restarting Mixxx...

Reply:
Fixed: 1a1953898513d207e0f52e16cb35e219bdc5b894